tools/tcprtt_example.txt

```diff
@@ -47,50 +47,103 @@ and remote address 192.168.122.100 and remote port 80.
 
 Tracing at server side, show each clients with its own histogram.
 For example, run tcprtt on a storage node to show initiators' rtt histogram:
-# ./tcprtt -i 1 -m --lport 3260 --byraddr
+# ./tcprtt.py -i 1 --lport 3260 --byraddr -v
```

Comment:
-e and update all examples below for option name and where to output additional stats.